lib/dataFile/controllers/authController.js

```diff
@@ -0,0 +1,229 @@
+"use strict";
```

Comment:
Add this this file to your .gitignore file

Reply:
Done